modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/Field/FieldStateless.es.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+import {PageProvider} from '../../hooks/usePage.es';
+import {useFieldTypesResource} from '../../hooks/useResource.es';
+import {Field} from './Field.es';
+
+export const FieldStateless = ({onBlur, onChange, onFocus, ...otherProps}) => {
```

Comment:
I think it will be very useful for me in the future for replacing my [calculatorDisplay](https://github.com/liferay-forms/liferay-portal/pull/28/files#diff-645afc789b08b53d038e7c5b004a38f0a354a90bb95ce5964ba527a3e6c5a5acR29)